yaas/config.py

```diff
@@ -6,7 +6,9 @@
 # These default configs are overriden based on
 # command line arguments or environment variables
```

Comment:
Sorry, one last thing. This comment is out of date--we don't support command line overrides.